client/blocks/jitm/templates/default.jsx

```diff
@@ -15,11 +15,14 @@ export default function DefaultTemplate( {
 	message,
 	description,
 	featureClass,
+	icon,
 	tracks,
 	trackImpression,
 	onClick,
 	onDismiss,
 } ) {
+	const isJetpack = icon && icon.indexOf( 'jetpack' ) !== -1;
```

Comment:
Is being fuzzy here (accepting `jetpackSomething` and `somethingjetpack`) on purpose?

Reply:
Yes. We're not using an exact match since, [due to the way the JITMs requests are implemented now](https://github.com/Automattic/jetpack/blob/c884a5beac7f0ff5122167c984bdd389aac13822/packages/jitm/src/class-jitm.php#L600), we're getting a markup instead of a simple `jetpack` string here.

Once the underlying JITM implementation changes, this could be updated to an exact match, though it would still work as expected.